data/remnant missions.txt

```diff
@@ -1331,6 +1331,26 @@ mission "Remnant Return the Samples Compatibility Patch"
 		fail
 
 
+# "This mission is intended to serve as a notification that one timer has elapsed and how long to wait until the missions start again."
+mission "Remnant: Return the Samples 3 Exploration Hint"
+	name "Return the Samples Timer Elapsed"
+	description "According to your commlink, Plume has reached a milestone in his research."
+	landing
+	source Aventine
+	to offer
+		has "event: remnant: return the samples timer"
+	on offer
+		event "remnant: wait for plume" 7 14
+		conversation
+			`As you disembark from <ship> you almost run into Plume as he strides across the landing area. "Captain <first>, I didn't see you there!" You glance back at the bulk of your ship that you just stepped out of and look at him curiously. His chant takes on a slightly apologetic tone "I've been lost in thought thinking about everything we have discovered lately, especially the most recent recordings you made. I still have a lot to analyze, though." He pauses, looking thoughtful. "That being said, I will probably have more work for you in a week or two."`
+			`	Plume asks a few more questions about what you have been doing, then politely excuses himself to continue his research. "Enjoy your explorations!" he trills to you as he departs.`
+			fail
+
+
+
+event "remnant: wait for plume"
```

Comment:
Should this be used anywhere, or is this for VS2?

Reply:
This is for VS2.